decidim-admin/app/controllers/decidim/admin/newsletters_controller.rb

```diff
@@ -156,6 +158,14 @@ def content_block_for_newsletter
 
         @content_block_for_newsletter ||= @newsletter.template
       end
+
+      def images_block_context
+        form(NewsletterForm).from_model(content_block).images
```

Comment:
![image](https://user-images.githubusercontent.com/491891/97420379-857f9e00-190b-11eb-96bf-67b127dcc7d9.png)

I like this!